README.md

```diff
@@ -110,7 +116,7 @@ Signatures for an auction with participation restriction can be created like tha
 export NETWORK='your network'
 export INFURA_KEY=INFURA_KEY_HERE
 export PK=PRIVATE_KEY_FOR_SIGNING
-yarn hardhat generateSignatures --auction-id "Your auctionId" --file-with-address "./your_address_inputs.txt" --network $NETWORK
+yarn hardhat generateSignatures --auction-id "Your auctionId" --file-with-address "./your_address_inputs.txt" --post-to-api true  --network $NETWORK
```

Comment:
This command is going to upload all the generated signatures on our services, and the user has no reason to know that. I'd remove it from the arguments and add a line below stating something like:
```
The generated signatures can be directly uploaded to the backend by adding the flag `--post-to-api true` to the previous command. Uploading signatures allows all authorized users to create orders from the web interface without the extra friction of managing a signature.
```